connect.go

```diff
@@ -153,6 +153,8 @@ func Connect(connectStr string) (success bool) {
 	// Wait for a clear channel
 	switch url.Scheme {
 	case MethodArdop:
+	case MethodVaraFM:
+	case MethodVaraHF:
```

Comment:
This is not how multiple cases are written in Go, and the code will not work as you expect. See https://go.dev/tour/flowcontrol/10.

The idiomatic way to specify multiple values for the same case would be `case MethodArdop, MethodVaraFM, MethodVARAHF:`.

I haven't checked, but if the VARA transport package implements the `transport.BusyChannelChecker` interface then this could work if you pass the correct TNC reference to the function as well.

Reply:
Ack.